### PR TITLE
Issue #3470

### PR DESCRIPTION
**Pull request by cristinelpopescu**
Add argmax function to numpy frontend

### CODE REVIEW

File: ivy/functional/frontends/numpy/sorting_searching_counting/sorting.py [hunk 1]
```diff
@@ -3,11 +3,21 @@
...
+        /,
+        *,
+        axis=-1,
+        kind=None,
+        order=None,
 ):
     return ivy.argsort(x, axis=axis)
+
+
+def argmax(x,
```
Comment (roboFiddle) [inline ivy/functional/frontends/numpy/sorting_searching_counting/sorting.py:16]: These should match the names and order of parameters of the matching NumPy method, as described [here](https://numpy.org/doc/stable/reference/generated/numpy.argmax.html). Specifically, the `x` parameter should be named `a`.

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_sorting.py [hunk 2]
```diff
@@ -53,3 +53,37 @@ def test_numpy_argsort(
...
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="numpy",
+        fn_tree="argmax",
+        input=np.asarray(x, dtype=input_dtype),
```
Comment (roboFiddle) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_sorting.py:85]: These keyword parameters should match the names of the frontend method. Thus, `input` should be renamed to `a`.

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_sorting.py [hunk 3]
```diff
@@ -53,3 +53,37 @@ def test_numpy_argsort(
...
+        axis=axis,
+        keepdims=False,
```
Comment (roboFiddle) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_sorting.py:87]: You should generate a parameter for `keepdims` using `st.booleans()` and use it here.

File: ivy/functional/frontends/numpy/sorting_searching_counting/sorting.py [hunk 4]
```diff
@@ -3,11 +3,21 @@
...
+           /,
+           *,
+           axis=None,
+           keepdims=False,
+           out=None,
+           ):
+    return ivy.argmax(x, axis=axis)
```
Comment (roboFiddle) [inline ivy/functional/frontends/numpy/sorting_searching_counting/sorting.py:23]: You should pass in the `keepdims` and `out` parameter here too.

Comment (cristinelpopescu) [reply ivy/functional/frontends/numpy/sorting_searching_counting/sorting.py:23]: I noticed that I wrote the function and the test in the wrong files, according to the ivy documentation, "argmax" is in the "searching" category. I will fix the mistake I made.

**Review by cristinelpopescu — COMMENTED** [review]
I add the function to 'searching' where it should have been from the beginning (not in sorting, because is a searching method).
I made the requested changes and passed my local test.

**Comment by roboFiddle** [comment]
Looks good, merging. Thanks for the contribution!

**Pull request merged**